content/docs/tasks/telemetry/export-logs-through-stackdriver.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Exporting Istio Access Logs to BigQuery, GCS, Pub/Sub through Stackdriver
+description: How to export Istio Access Logs to different sinks like BigQuery, GCS, Pub/Sub through Stackdriver.
+weight: 70
+---
+
+This task shows how to export Istio Access Logs to different sinks like
+BigQuery, GCS, Pub/Sub through Stackdriver A user can do analytics on data
```

Comment:
Suggestion:
> Analytics may be performed on Istio data by flowing logs from Istio to Stackdriver and exporting those logs to various configured sinks such as BigQuery (link), GCS (link), or Cloud Pub/Sub (link).

Reply:
Changed to "This task shows how Istio data using logs can flow from Istio to [Stackdriver](https://cloud.google.com/stackdriver/)
and export those logs to various configured sinks such as such as
[BigQuery](https://cloud.google.com/bigquery/), [Google Cloud Storage(GCS)](https://cloud.google.com/storage/)
or [Cloud Pub/Sub](https://cloud.google.com/pubsub/). At the end of this task you can perform
analytics on Istio data from your favorite places such as BigQuery, GCS or Cloud Pub/Sub." 
As all other tasks are structured this way.. please see if this looks ok now?